src/api/api.py

```
@@ -0,0 +1,112 @@
+        elif clean == "raw+clean":
+
+            results = await get_message_use_case.get_raw_clear(query, chat_id, k)
+        elif clean == "raw":
+
+            results = await get_message_use_case.get_raw(query, chat_id, k)
+            
+        if not results:
+            return {"message": "Ничего не найдено", "results": []}
+            
+        return {"results": results}
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)